src/views/hostnetworkconfiguration/components/TopologySidebar/InterfaceDrawer/NeighborInformations.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import {
+  CHASSIS_ID,
+  NodeNetworkConfigurationInterfaceLLDPNeighbor,
+} from 'src/nmstate-types/custom-models/NodeNetworkConfigurationInterfaceLLDP';
+
+import { Stack, StackItem, Tooltip } from '@patternfly/react-core';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+import {
+  getChassisId,
+  getDescription,
+  getIee8021Vlans,
+  getPortId,
+  getSystemDescription,
+  getSystemName,
+} from '@utils/neighbors/getters';
+
+const NeighborInformations: FC<{ neighbor: NodeNetworkConfigurationInterfaceLLDPNeighbor }> = ({
```

Comment:
nitpick: 'Information' is a plural noun which means you don't need the 's' at the end.

Reply:
This still needs to be changed.